test/specs/routing.test.js

```diff
@@ -533,6 +533,17 @@ module.exports = (fetchMock) => {
 				expect(fm.calls(true).length).to.equal(1);
 			});
 
+			it('match relative urls with dots', async () => {
```

Comment:
given that the src branches depending on whether it's an absolute or relative url, I'd like to see a test for each case